crates/bevy_render/src/camera/camera.rs

```diff
@@ -61,6 +62,48 @@ impl Camera {
         let screen_space_coords = (ndc_space_coords.truncate() + Vec2::ONE) / 2.0 * window_size;
         Some(screen_space_coords)
     }
+
+    /// Given a position in screen space, compute the world-space line that corresponds to it.
+    pub fn screen_to_world_line(
```

Comment:
Both of these functions feel like they should be a method on the Camera, not a function that takes a camera as the argument.

Reply:
See: #1258

Reply:
I wrote it like that originally, but it felt weird that it was a method on the camera but still required the camera's global transform. I thought it was cleaner as a function but I'm happy to change it if you think it's better or more consistent that way (I'm not sure how this is normally handled in bevy).